CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - Change the GUI of text input field box. [#991](https://github.com/rokwire/events-manager/issues/991)
 - Update the style of pagination in user event listing page. [#989](https://github.com/rokwire/events-manager/issues/989)
+- Update Login and Logout button. [#1009](https://github.com/rokwire/events-manager/issues/1009)
```

Comment:
Please add the new entry to the end of the section as we discussed last week. Thanks.